aptos-move/framework/aptos-framework/sources/transaction_validation.spec.move

```diff
@@ -31,19 +31,19 @@ spec aptos_framework::transaction_validation {
     /// Ensure caller is `aptos_framework`.
     /// Aborts if TransactionValidation already exists.
     spec initialize(
-        aptos_framework: &signer,
-        script_prologue_name: vector<u8>,
-        module_prologue_name: vector<u8>,
-        multi_agent_prologue_name: vector<u8>,
-        user_epilogue_name: vector<u8>,
-   ) {
+    aptos_framework: &signer,
```

Comment:
Why spec formatting looks like this?🧐

Reply:
I was wondering too, seems some formatter issue